### DISCUSSION TITLE
Request for Reinstatement of Suspended GitHub Account

### DISCUSSION THREAD

**Discussion by SmitBog** [community/community#134383] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

General

### Body

I hope this message finds you well. I am writing to request the reinstatement of my GitHub account, which was recently suspended. My account username is **sumitboghara**, and the email associated with the account is **sumitboghara.vision@gmail.com**

I have already emailed your support account but have not yet received a response. My work is pending due to this suspension, and I urgently need assistance to reactivate my account.

Ticket Id : 2897688

Thank you for your prompt attention to this matter.

**Comment by SAHELISAHAA** [comment]
@SmitBog when it got suspended?

**Reply by SmitBog** [reply]
@SAHELISAHAA  around 2 weeks ago

**Comment by queenofcorgis** [comment, marked as answer]
Hi @SmitBog 

Thank you for your patience. The community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding!

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!
